aws-rds-dbinstance/aws-rds-dbinstance.json

```diff
@@ -384,7 +384,6 @@
     "/properties/MasterUsername",
     "/properties/NcharCharacterSetName",
     "/properties/Port",
-    "/properties/PubliclyAccessible",
```

Comment:
Doesn't look like this property has been added to the Translator yet:
https://github.com/aws-cloudformation/aws-cloudformation-resource-providers-rds/blob/master/aws-rds-dbinstance/src/main/java/software/amazon/rds/dbinstance/Translator.java#L311